### ISSUE TITLE
Scoring Parameter of DRT: Which Cost is the Right One? Do I need to Consider the marginalUtilityOfTraveling_hr

### ISSUE THREAD

**Issue by jingjunL** [matsim-org/matsim-code-examples#933]
I had a DRT run with our Brussels scenario, and I set some fare rules under the config section `drtfare`. For instance, I have set the value `timeFare_h` to 2.

However, when I opened the output_config, I saw under the `planCalScore` - `scoringParameters` that the default `marginalUtilityOfTraveling_hr` for drt is already -6. This means if the `marginalUtilityOfTraveling_hr` for car is 0, people who use drt will already spend 6 euros in addition when travelling via drt. And with the timeFare that I have set, it will become 8, which means drt will be very less attractive compared to a car.

I am wondering whether my understanding is correct? So if I really want to set the "real" timeFare of drt to 2 euro more compared to car, should I set that value instead to -4, which means -6 - (-2) . This is a bit confusing, can I have some clarification?

Thanks :)

**Comment by tschlenther** [comment]
Hi Jingjun :)
Well, the `marginalUtilityOfTraveling_hr` for car depends on your base case calibration. Usually, it should be negative. Is your marginal utility of travel distance also 0 for cars?
If yes, have you checked for distance distributions for car (or all modes) during calibration?

Second, trying to answer your question: It depends on what you want to model. If you want to model an explicit _monetary_ _fare_ that is imposed on users, you should rather do by using the drtFare. That is because the scoring parameters are for tuning the mode choice behavior, i.e. to model the _perception_ of traveling with a certain model. If, opposed to a specific monetary fare, you want to model that people _perceive_ travel time in a drt vehicle to be 2 euros more expensive than the travel time in the car, you can indeed put it into the scoring parameters. Make sure to account for the `marginalUtilityOfMoney`, correctly.
I think there is some literature available on the perception of (travel time in) drt. However, the relation to car and/or the absolute value you should put into your model for the `marginalUtilityOfTraveling_hr` for drt in case you want to model a fare depends on your calibration. If you have for example an idea of how the distance distribution of the drt trips should look like, you can use that parameter to calibrate your drt trips.

**Comment by jingjunL** [comment]
> hat is because the scoring parameters are for tuning the mode choice behavior, i.e. to model the _perception_ of traveling with a certain model. If, opposed to a specific monetary fare, you want to model that people _perceive_ travel time in a drt vehicle to be 2 euros more expensive than the travel time in the car, you can indeed put it into the scoring parameters. Make sure to account for the `marginalUtilityOfMoney`, correctly.

Hi @tschlenther :)

To your question 1: yes, our model is calibrated similarly to the Berlin scenario, the marginal utility of travel of `car` is set to 0, and other modes are set accordingly based on this car value (i.e. pt is less comfortable than the car, so the marginal utility of hour will be lower). For car mode, the marginal utility of travel distance is not 0. I set it based on the fare of patrol. And indeed, our model is calibrated against mode share, distance share and some count data. Overall, very similar to what you did for Berlin/Hamburg 

For question 2: so if I understand the module `drtfare` correctly, the actual travel disutility drt agents perceived is calculated as:

`marginalUtility_hr_drt * drtTravelTime + marginalUtilityOfMoney * drtTimeFare_h * drtTravelTime + marginalUtility_km_drt * drtTravelDistance + marginalUtilityOfMoney * drtDistanceFare_m * drtTravelDistance`

In the case where we have the minFarePerTrip, if:

`drtTimeFare_h * drtTravelTime + drtDistanceFare_m * drtTravelDistance < minFarePerTrip`, 

Accordingly, the disutility will be calculated as follows:

`marginalUtility_hr_drt * drtTravelTime + marginalUtility_km_drt * drtTravelDistance + marginalUtilityOfMoney * minFarePerTrip`

So, in other words, the `drtFare` is just a separate calculation system added to the original drt travel disutility calculated with the scoring parameters. Am I correct?

Thanks!
